ci-operator/step-registry/telco5g/release-cluster/telco5g-release-cluster-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: telco5g-release-cluster
+  from_image:
+    namespace: ci
+    name: telco-bastion
+    tag: latest
+  commands: telco5g-release-cluster-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 500Mi
```

Comment:
these numbers look exaggerated
probably can do with 
`cpu: 100m`
and not sure what's the minimum memory for python

it is better to lower these numbers to avoid running out of cluster resources

Reply:
I think probably it's worth to check separately, I'm using numbers that were there before and proved to work.